### DISCUSSION TITLE
add Ckeditor

### DISCUSSION THREAD

**Discussion by minhkhoablieu** [filamentphp/filament#502] [Ideas]
Hello! I would love to use Filament for my project. But after using it for a while, I found that there is an inadequacy that is editing the html of the content in the RichEditor for example adding a class to the div but it is not possible. 

Can you change it with Ckeditor or add some features?

**Comment by danharrin** [comment]
If you wish, you can create a custom Ckeditor form component, but this field type is not in our development plans.
